openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -16,7 +16,7 @@
     "build": "webpack --config ./webpack.config.prod.js --env production",
     "postinstall": "yarn run build-check",
     "preinstall": "cd ../../../../.. && yarn install --frozen-lockfile",
-    "pre-commit": "lint-staged --concurrent false",
+    "pre-commit": "lint-staged --concurrent false && yarn generate:app-docs",
```

Comment:
If you wish it to run auto consider moving this to
https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-ui/src/main/resources/ui/.husky/pre-commit